_locales/en/messages.json

```diff
@@ -367,8 +367,8 @@
       }
     },
     "submitDebugLog": {
-      "message": "Submit debug log",
-      "description": "Menu item and header text for debug log modal, title case."
+      "message": "Debug log",
```

Comment:
Where is this string used in the app? Can we remove it?

Reply:
It’s used as title of the debug log modal. The idea is to match the menu bar entry and the modal title for consistency but we currently use *Title Case* for menus and *Sentence case* for in-app titles, hence the two strings.